wildlifelicensing/apps/customer_management/templates/wl/officer_edit_customer_details.html

```diff
@@ -4,13 +4,23 @@
 
 {% load bootstrap3 %}
 
+{% block requirements %}
+    require(['jQuery', 'js/officer_edit_customer_details'], function($, module) {
+    $(function(){
+    module.init();
+    });
+    });
```

Comment:
Yes every time I do a auto-format everything the the requirements block is whacked.